src/ui/input-range.js

```diff
@@ -31,7 +33,11 @@ export default function InputRange(options = {}) {
     },
     render() {
       return `
-      <input id="${this.getId()}" type="range" min="${minValue}" max="${maxValue}" value="${initialValue}" step="${step}" class="${cls}" style="${style}">
+      <div class="flex no-wrap text-small align-middle">
+        <input id="${this.getId()}" type="range" min="${minValue}" max="${maxValue}" value="${initialValue}" step="${step}" class="${cls}" style="${style}" oninput="this.nextElementSibling.value = this.value">
+        <output class="padding-left-small">${initialValue}</output><div>${unit}</div>
+      </div>
+      <div class="text-small text-align-center padding-smallpadding-top-smallest width-full">${label}</div>
```

Comment:
Here be a typo, a missing space. Might wanna fix that.

Reply:
Still grumpy I see 😉

Reply:
Always.